rllib/tuned_examples/dreamerv3/atari_200M.py

```diff
@@ -50,10 +50,21 @@
             "full_action_space": False,
             # Already done by MaxAndSkip wrapper: "action repeat" == 4.
             "frameskip": 1,
-        }
+        },
+    )
+    .env_runners(
+        num_env_runners=(args.num_env_runners or 0),
+        # If we use >1 GPU and increase the batch size accordingly, we should also
+        # increase the number of envs per worker.
+        num_envs_per_env_runner=8 * (args.num_gpus or 1),
+        remote_worker_envs=True,
+    )
+    .learners(
+        num_learners=0 if args.num_gpus == 1 else args.num_gpus,
```

Comment:
Btw, isn't it possible to use here fractional GPUs as well? Even though the model is large I guess a V100 could be shared between 2 learners?